autogpt_platform/backend/backend/util/service.py

```diff
@@ -167,14 +166,12 @@ def run(self):
             redis.connect()
         if self.use_rabbitmq:
             logger.info(f"[{self.__class__.__name__}] ⏳ Configuring RabbitMQ...")
-            if self.use_async:
-                self.rabbitmq_service = rabbitmq.AsyncRabbitMQ(self.use_rabbitmq)
-                self.shared_event_loop.run_until_complete(
-                    self.rabbitmq_service.connect()
-                )
-            else:
-                self.rabbitmq_service = rabbitmq.SyncRabbitMQ(self.use_rabbitmq)
-                self.rabbitmq_service.connect()
+            # if self.use_async:
+            self.rabbitmq_service = rabbitmq.AsyncRabbitMQ(self.use_rabbitmq)
+            self.shared_event_loop.run_until_complete(self.rabbitmq_service.connect())
+            # else:
+            #     self.rabbitmq_service = rabbitmq.SyncRabbitMQ(self.use_rabbitmq)
+            #     self.rabbitmq_service.connect()
```

Comment:
building out this interface is painful until I know exactly what it looks like so we are using only async right now so I don't have to keep updating interfaces twice that I'm not using. FInal PR will try to have this feature parity, but it may not

Reply:
Is there a way we can track some of these things so when the final PR comes up we can check none have been forgotten.

Maybe a GitHub issue?

Reply:
I'm attaching them to linear project right now as you mention them